package.json

```
@@ -124,39 +131,61 @@
     ]
   },
   "scripts": {
+    "ci:test": "run-s test:*",
     "dep-check": "aegir dep-check src/**/*.ts test/**/*.ts generated/**/*.ts",
     "fix": "run-s fix:*",
+    "fix:gen": "./npm-scripts/fix/gen.sh",
     "fix:lint": "aegir lint --fix",
     "lint": "run-s lint:*",
     "lint:main": "aegir lint",
-    "lint:project": "check-aegir-project",
+    "lint:ts": "aegir ts -p check -- -p tsconfig.json --suppress 6133@generated 6192@generated --stats",
+    "lint-TODO:project": "check-aegir-project # currently broken due to corrupting the repoUrl",
     "release": "aegir release",
-    "postinstall": "run-s gen",
-    "build": "aegir build",
+    "postinstall": "./npm-scripts/postinstall.sh",
+    "build": "aegir build -- -p tsconfig.json --suppress 6133@generated 6192@generated --stats",
+    "build:main": "aegir build -- -p tsconfig.json --suppress 6133@generated 6192@generated --stats",
+    "build:docs": "aegir ts docs",
     "test": "run-s test:*",
+    "test:browser": "aegir test --target browser",
+    "test:webworker": "aegir test --target webworker",
     "test:electron": "aegir test --target electron-main",
     "test:node": "aegir test --target node --cov && npx nyc report",
-    "pregen:node": "openapi-generator-cli validate -i https://raw.githubusercontent.com/ipfs/pinning-services-api-spec/main/ipfs-pinning-service.yaml",
-    "gen": "run-p gen:*",
+    "pregen": "openapi-generator-cli validate -i https://raw.githubusercontent.com/ipfs/pinning-services-api-spec/main/ipfs-pinning-service.yaml",
```

Client should be deterministic – we should build against specific [API version](https://github.com/ipfs/pinning-services-api-spec/releases):
```suggestion
    "pregen": "openapi-generator-cli validate -i https://raw.githubusercontent.com/ipfs/pinning-services-api-spec/v1.0.0/ipfs-pinning-service.yaml",
```